modules/ROOT/pages/troubleshooting-tips.adoc

```diff
@@ -47,108 +116,115 @@ docker logs my-container > log.txt
 ----
 
 Keep in mind that Docker logs are deleted if the container is recreated.
+--
+======
 
 === Message levels
 
-Some log messages contain text that sounds like an error, but they are not errors.
-The message's `level` typically indicates severity:
+Some log messages contain text that seems like an error but they aren't errors.
+Instead, the message's `level` indicates severity:
 
-* `level=debug` and `level=info`: Expected and normal messages that are typically not errors.
-However, if you enable `DEBUG` logging, `debug` messages can help you find the source of a problem.
+* `level=debug` and `level=info`: Expected and normal messages that typically aren't errors.
++
+If you enable `DEBUG` logging, the `debug` messages can help you find the source of a problem by providing information about the environment and conditions when the error occurred.
 
-* `level=warn`: Reports an event that wasn't fatal to the overall process, but could indicate an issue with an individual request or connection.
+* `level=warn`: Reports an event that wasn't fatal to the overall process but might indicate an issue with an individual request or connection.
 
 * `level=error`: Indicates an issue with {product-proxy}, the client application, or the clusters.
 These messages require further examination.
 
-If the meaning of a `warn` or `error` message isn't clear, you can submit an issue in the {product-proxy-repo}/issues[{product-proxy} GitHub repository].
+If the meaning of a `warn` or `error` message isn't clear, you can <<report-an-issue,report an issue>>.
```

Comment:
Is there any self-serve we can direct people to before telling them to report an issue?